dump/offload_manager.cpp

```diff
@@ -17,25 +17,11 @@ OffloadManager::OffloadManager(sdbusplus::bus::bus& bus,
         _bus, _dumpQueue, bmcEntryIntf, bmcEntryObjPath, DumpType::bmc);
     _offloadHandlerList.push_back(std::move(bmcDump));
 
-    // add host dump offload handler to the list of dump types to offload
```

Comment:
probably we should have all these handlers as we are supporting bmc, hostboot, hardware, and sbe dump offloads